src/mame/drivers/galaxian.cpp

```diff
@@ -6534,6 +6623,25 @@ MACHINE_CONFIG_START(galaxian_state::amigo2) // marked "AMI", but similar to abo
 	AY8910(config, m_ay8910[1], 12_MHz_XTAL / 2 / 2 / 2).add_route(ALL_OUTPUTS, "speaker", 0.25); /* matches PCB, needs verification */
 MACHINE_CONFIG_END
 
+MACHINE_CONFIG_START(galaxian_state::frogf2) // origins unknown, but is akin to other bootlegs with Sidam(?) origin
+	sidam_bootleg_base(config);
+
+	// alternate memory map
+	m_maincpu->set_addrmap(AS_PROGRAM, &galaxian_state::frogf2_map);
+
+	// 2nd CPU to drive sound
+	MCFG_DEVICE_ADD("audiocpu", Z80, 12_MHz_XTAL / 2 / 2 / 2) /* verified on PCB */
+	MCFG_DEVICE_PROGRAM_MAP(frogger_sound_map)
+	MCFG_DEVICE_IO_MAP(frogger_sound_portmap)
+	GENERIC_LATCH_8(config, m_soundlatch);
```

Comment:
It would be better to separate this from the stuff directly related to the audio CPU wiht a blank line to keep things clearer.